src/main/java/uk/gov/hmcts/reform/cwrdapi/domain/CaseWorkerLocation.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.reform.cwrdapi.domain;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+
+@Entity(name = "case_worker_location")
+@Getter
+@Setter
+@NoArgsConstructor
+@EntityListeners(AuditingEntityListener.class)
+@SequenceGenerator(name = "case_worker_location_id_seq",
```

Comment:
I think sequence not needed in many tables apart from main profile table. As it serial and postgress take care of it.

Reply:
Sure will remove sequence. thanks